.travis.yml

```diff
@@ -65,21 +65,21 @@ before_script:
   - >
      cmake \
        -DCMAKE_BUILD_TYPE=${BUILD_TYPE} \
-       -DOpenMVG_BUILD_TESTS=ON \
-       -DOpenMVG_BUILD_EXAMPLES=ON \
-       -DOpenMVG_USE_OPENCV=ON \
+       -DOPENMVG_BUILD_TESTS=ON \
```

Comment:
@gregoire-dl Sorry I merged the PR but only then I realised that the README should also be updated.
I think only the "CMake options" section needs to be updated https://github.com/alicevision/openMVG/blob/popart_develop/INSTALL.md#cmake-options, mostly it should be a matter of changing `OpenMVG_*` --> `OPENMVG_*`.